quickfixj-core/src/main/java/quickfix/field/converter/CharArrayConverter.java

```diff
@@ -0,0 +1,40 @@
+package quickfix.field.converter;
+
+import quickfix.FieldConvertError;
+
+/**
+ * Converts between character array and string.
+ */
+public class CharArrayConverter {
+
+    private static final String CHAR_ARRAY_REGEX = "[A-Za-z0-9]( [A-Za-z0-9])*";
+
+    public static String convert(char... chars) {
+        if (chars.length == 0) {
+            throw new IllegalArgumentException("empty character array");
+        }
+
+        StringBuilder builder = new StringBuilder(chars.length * 2 - 1);
+        builder.append(chars[0]);
```

Comment:
I am asking myself if we could use `String.join()` or respectively `StringJoiner` here.  So we would not need the loop and the code would be cleaner. What do you think?

Reply:
OK, just realized that we would need to convert the `char` to `CharSequence` or wrap it in `String.valueOf( char )'.

Reply:
I was thinking about the same, but `StringBuilder` was better.

Reply:
Because of the needed char to String conversion? Or because of other reasons?

Reply:
Yes - because `java.util.StringJoiner#add` requires `java.lang.CharSequence`